requirements/local-inference.in

```diff
@@ -0,0 +1 @@
+unstructured-inference>=0.2.1
```

Comment:
Do we want to put this as an optional requirement in `setup.py` so you can do `pip install unstructured[local_inference]`?

Reply:
Yep. Added in [8b6d379](https://github.com/Unstructured-IO/unstructured/pull/125/commits/8b6d379b215310801ccc516f47945f3670d71a51).